### PR TITLE
Fix the compiler error with fips

### LINKED ISSUE #3303: Can't compile with --enable-fips option
```
In file included from ink_code.cc:27:
../../lib/ts/INK_MD5.h:34:3: error: unknown type name 'MD5_CTX'
  MD5_CTX _ctx;
```

### PR DESCRIPTION
**Pull request by scw00**
Fix #3303

### CODE REVIEW

**Comment by maskit** [comment]
This doesn't look like a right approach. It allows using MD5. From my understanding, the point of the `#if` in ink_code.{c|hh} is ensuring that MD5 is not available at all on main programs (traffic_server, traffic_manager, etc.).

It seems like `ink_code_md5` was copied into `jtest.cc` on #2798 so that test programs can use MD5 even if `--enable-fips` is specified, but the function was removed on #3236.

**Comment by scw00** [comment]
@maskit  updated! Can you please review again ?

File: tools/jtest/jtest.cc [hunk 1]
```diff
@@ -3443,19 +3443,16 @@ seen_it(char *url)
...
   char *para = strrchr(url, '#');
   if (para) {
     l = para - url;
   } else {
     l = strlen(url);
   }
-  ink_code_md5((unsigned char *)url, l, u.md5);
-  uint64_t x = u.i[0] + u.i[1];
+  CryptoHash hash;
+  CryptoContext().hash_immediate(hash, reinterpret_cast<void *>(url), l);
```
Comment (maskit) [inline tools/jtest/jtest.cc]: Looks OK, but did you confirm that other parts don't expect MD5?

File: tools/jtest/jtest.cc [hunk 2]
```diff
@@ -3443,19 +3443,16 @@ seen_it(char *url)
...
+  uint64_t x = hash.u64[0] + hash.u64[1];
```
Comment (SolidWallOfCode) [inline tools/jtest/jtest.cc]: Shouldn't this be `uint64_t x = hash.fold();` in case other hashes want to fold differently?

**Comment by scw00** [comment]
@SolidWallOfCode Updated! Can you please review again?

File: tools/jtest/jtest.cc (hunk 1 above)
Comment (maskit) [reply tools/jtest/jtest.cc]: After #2798, I fixed several array sizes on #3082. I meant something like that.

**Comment by maskit** [comment]
@SolidWallOfCode I believe you.

**Pull request merged**